Updated the text for unsupported hardware

spolifroni-amd opened the pull request.

spolifroni-amd committed:
Updated the text for unsupported hardware with wording approved by PM

peterjunpark commented:
@spolifroni-amd 

>✅: **Supported** - Official software distributions of the current ROCm release fully enables this hardware.

Could you fix the s-v agreement? I missed this in the first PR. "enables" --> "enable" or "support"

spolifroni-amd committed:
Changed "enables" to "support"

Original:  Official software distributions of the current ROCm release fully enables this hardware.

Updated:  Official software distributions of the current ROCm release fully support this hardware.

spolifroni-amd commented:
> @spolifroni-amd
> 
> > ✅: **Supported** - Official software distributions of the current ROCm release fully enables this hardware.
> 
> Could you fix the s-v agreement? I missed this in the first PR. "enables" --> "enable" or "support"

Done.

peterjunpark reviewed (approved):
lgtm
